src/Shared/Components/SelectPicker/SelectPicker.component.tsx

```diff
@@ -399,25 +446,11 @@ const SelectPicker = <OptionValue, IsMulti extends boolean>({
                                 createOptionPosition="first"
                                 onCreateOption={handleCreateOption}
                                 renderMenuListFooter={!optionListError && renderMenuListFooter}
+                                inputValue={inputValue}
+                                onInputChange={onInputChange}
```

Comment:
if this is required then this should be added for single select as well

Reply:
For now, It was required only in case of multi select